### PR TITLE
feat: update headers

### PR DESCRIPTION
**Pull request by goldenmaya**
### Jira-link:

https://virtocommerce.atlassian.net/browse/VCST-606
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.55.0-pr-1009-e0e0-e0e05b3a.zip

### CODE REVIEW

File: client-app/ui-kit/components/atoms/typography/vc-typography.vue [hunk 1]
```diff
@@ -1,93 +1,167 @@
...
+  if (props.textTransform) {
+    return props.textTransform;
+  }
+
+  if (isHeader.value) {
+    return "uppercase";
+  }
+
+  return "none";
 });
```
Comment (asvishnyakov) [inline client-app/ui-kit/components/atoms/typography/vc-typography.vue:70]: I'm not sure this is good for extensibility, May be it's better to do it in styles instead?

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: client-app/ui-kit/components/atoms/typography/vc-typography.types.d.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+declare global {
+  type VcTypographyVariantType = "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "base";
+  type VcTypographyWeightType = "thin" | "light" | "normal" | "bold" | "black";
+  type VcTypographyTextTransformType = "uppercase" | "lowercase" | "capitalize" | "none";
+}
+
+export {};
```
Comment (asvishnyakov) [inline client-app/ui-kit/components/atoms/typography/vc-typography.types.d.ts:7]: You was needed to add `export {}` to tell Typescript it's a module, if there is not `import`

File: client-app/shared/catalog/components/category.vue [hunk 3]
```diff
@@ -59,7 +59,7 @@
       <!-- Content -->
       <div ref="contentElement" class="grow">
         <div class="flex">
-          <h2 class="text-21 font-bold uppercase text-gray-800 lg:my-px lg:text-25 lg:leading-none">
+          <VcTypography tag="h1">
```
[the remark this answers is not in the document]
Comment (goldenmaya) [reply client-app/shared/catalog/components/category.vue:62]: there is normal-case there

File: client-app/assets/styles/main.scss [hunk 4]
```diff
@@ -2,59 +2,20 @@
...
-@import "helpers";
-@import "print";
-@import "vendors";
-
 @layer base {
-  body {
-    @apply text-[--color-body-text];
+  html, :host {
+    @apply font-lato;
   }
```
Comment (asvishnyakov) [inline client-app/assets/styles/main.scss:8]: I found that there is difference in classes on body in `liquid`, used for production (environments) and `index.html`, used for local development (and storefront-less configuration in future). `font-lato` is missed on first:
https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/ec59c90530c6dbf5340478c70d371b3bd5c8fe4d/layout/theme.liquid#L9
https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/ec59c90530c6dbf5340478c70d371b3bd5c8fe4d/index.html#L16
I wanted to make PR and bring it into `index.html`, but than found you did this change and now I propose to remove it from `liquid` in this PR instead.

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**